doc/sphinx/language/gallina-specification-language.rst

```diff
@@ -1503,19 +1503,18 @@ Each attribute has a name (an identifier) and may have a value.
 A value is either a :token:`string` (in which case it is specified with an equal ``=`` sign),
 or a list of attributes, enclosed within brackets.
 
-Currently,
-the following attributes names are recognized:
+Currently, the following attributes names are recognized:
 
-``monomorphic``, ``polymorphic``
-    Take no value, analogous to the ``Monomorphic`` and ``Polymorphic`` flags
-    (see :ref:`polymorphicuniverses`).
+``universes(monomorphic)``, ``universes(polymorphic)``
+    Take no value, equivalent to the ``Monomorphic`` and
+    ``Polymorphic`` flags (see :ref:`polymorphicuniverses`).
```

Comment:
I'd rather say that the `universes` attribute must take as value one of the optional sub-attributes `monomorphic`, `polymorphic`, `template`, `notemplate`.

Reply:
Talking about template in this section seems wrong, it's pretty specific to Inductive/Record.

Reply:
IMO this section must present all the attributes that exist, and limit their scope by saying on which commands they apply. But also OK for merging as is if you want.

Reply:
I changed the wording a bit to be closer to how I think of this stuff, have a look.